### DISCUSSION TITLE
Should we try to continue lagon?

### DISCUSSION THREAD

**Discussion by anbraten** [lagossapp/lagoss#2] [General]
Hey, I am a big fan of the project [Tom](https://github.com/QuiiBz) created and would like to continue development on it.

As the official hosting over at [lagon.app](https://lagon.app/) will end on 1st of March 2024 this is an approach to continue the project somehow. 

There is currently a public testing instance deployed at <https://app.lagoss.com>. However it should be only used for basic testing and data might be deleted at any point. If you want a proper setup, please deploy your own instance => [Install my own instance](https://docs.lagoss.com).

I would be happy if some people will help continuing & improving this fork of Lagon 🎉

**Comment by toinbis** [comment]
Hey @anbraten. Was having same idea in mind. Let's connect and discuss?

**Reply by anbraten** [reply]
Hey @toinbis, nice to see some more interest in keeping the project alive. For now I've started to fix some parts to be able to deploy an instance over at <https://app.lagoss.com>. Currently the instance is just using just a single serverless instance, but for my use-cases this is currently totally fine. Custom domains are not yet supported by my deployment as I have to tell the webserver (caddy) about it to fetch ssl certificates. I also started to update the dashboard "a bit" in #11.

**Comment by sohamdeogaonkar** [comment]
Great
I have tried to host it on own vps server.
I am unable to host that

I don't have aws 
So I don't what should I add at S3 Bucket

Could you guide me?

**Reply by anbraten** [reply]
You should be able to run it pretty easily with docker-compose: https://github.com/lagossapp/lagoss/blob/main/docker-compose.yml
Inside of that docker-compose there is minio which is a s3 compatible storage system.

**Reply by sohamdeogaonkar** [reply]
@anbraten Unable to Login Through Github

**Reply by anbraten** [reply]
@sohamdeogaonkar Should be working again.

**Reply by sohamdeogaonkar** [reply]
@anbraten Cant see my functions or create new one

**Reply by sohamdeogaonkar** [reply]
@anbraten It down.
Just to inform.
I am not using.

But I want to just contact you.
Any Social Media(Preferably telegram) Handle where we can contact?

**Reply by anbraten** [reply]
@sohamdeogaonkar Thanks. Planetscale has some inactivity system which kicked in again.

**Reply by anbraten** [reply]
You can write me here:
Twitter: https://twitter.com/anbraten
Matrix: @anbraten:matrix.org
Discord: anbraten
